### PR TITLE
Most likely almost certainly fixes the Holy Water+Potassium reaction not playing the hallelujah sound effect.

### LINKED ISSUE #28983: Holy WaterPot doesn't play a Hallelujah as advertised
(WEB REPORT BY: eneocho REMOTE: 206.221.180.138:7777)
# Revision
ff0589ffdc31642dae69da994d838a293aabb3c0

# Description
the tin. it says things. what the tin says.
# Steps to Reproduce
put holy water in contact with potassium
# What you Expected
to be able to RP as the worms from the worms game
# What Actually Happened
no hallelujah for me.

### PR DESCRIPTION
**Pull request by Eneocho**
[bugfix]
Closes #28983.
:cl:
 * bugfix: Holy Water/Potassium now correctly plays the Hallelujah sound effect.

### CODE REVIEW

File: code/modules/reagents/Chemistry-Recipes.dm [hunk 1]
```diff
@@ -95,7 +95,7 @@
 	
 /datum/chemical_reaction/explosion_potassium/holy/on_reaction(var/datum/reagents/holder, var/created_volume)
 	..()
-	playsound('sound/misc/holyhandgrenade.ogg', 100, 1)
+	playsound(src, 'sound/misc/holyhandgrenade.ogg', 100, 1)
```
Comment (jknpj) [inline code/modules/reagents/Chemistry-Recipes.dm]: maybe choosing the holder as source is better?

Comment (Eneocho) [reply code/modules/reagents/Chemistry-Recipes.dm]: I literally added that line originally on kurfursten's suggestion so I don't know.

Comment (Kurfursten) [reply code/modules/reagents/Chemistry-Recipes.dm]: jknpj is right, use the holder

Comment (Eneocho) [reply code/modules/reagents/Chemistry-Recipes.dm]: how though

Comment (Kurfursten) [reply code/modules/reagents/Chemistry-Recipes.dm]: instead of `src`, use `holder.my_atom`

Comment (Eneocho) [reply code/modules/reagents/Chemistry-Recipes.dm]: Ok, on it.

**Comment by Exxion** [comment]
You know, it doesn't have to be hopefully. You can test it.

**Pull request merged**